_topics/en/op_checktemplateverify.md

```diff
@@ -131,30 +131,230 @@ see_also:
+  been [proposed][tluv] several [times][op_evict], although some of
+  their authors have mentioned that CTV would be complementary to their
+  proposals. <!-- TODO:add citation; in particular I (harding) recall AJ saying this about TLUV -->
+
+- **Vaults:** without CTV or another consensus change, the only known way
+  to create a [vault][topic vaults] is using ephemeral keys and
+  presigned transactions that give spends to the vault's recovery
+  address priority over spends to a third-party.  Any ephemeral keys used
+  in the vault setup need to be securely deleted.  This approach is
+  fragile at best, making it reasonable for anyone with a large amount of
+  funds---the ideal vault user---to be wary of trusting presigned vaults
+  with large amounts of money.  By comparison, a CTV vault can replace
+  each use of ephemeral keys with a CTV output, which can be entirely
+  precomputed without any special key handling considerations.  The
+  result is still less dynamic than many users might like, so specific
+  consensus changes for vaults have been proposed, with at least one of
+  those proposals ([BIP345][] `OP_VAULT`) using CTV in some of its
+  operations.
+
+- **DLC efficiency:** [discreet log contracts][topic dlc] (DLCs) often
+  involve a large number of potential outcomes; for example, "what price
+  will Bitcoin be in USD terms at the end of the day, rounded to the
+  nearest whole dollar?" might easily involve more than 10,000 different
+  outcomes.  Using existing DLC constructions, each outcome requires
+  a separate [signature adaptor][topic adaptor signatures] that must be
+  created, exchanged, verified, and stored.  By comparison, CTV can
+  replace the adaptors and allow each party to independently compute
+  most of the state from a short specification of the
+  contract---minimizing bandwidth and storage.  A few signatures and
+  adaptors are still needed, but the vast majority are eliminated.
+
+## Alternatives to CTV
+
+Many alternatives to CTV have been proposed.  The following have been
+covered by Optech:
+
+- **Presigned transactions:** anything CTV can do can also be done using
+  presigned transactions and secure deletion of ephemeral keys.
+  However, this requires perfectly reliable backups and deletion.  For
+  CTV, deletion is not required and the amount of data that needs to be
+  reliably backed up consists of only the data a user expects to put
+  onchain anyway (some of which it may be possible to compute as needed).
+
+- **SIGHASH_ANYPREVOUT:** this [proposal][topic sighash_anyprevout],
+  which was designed to [improve payment channels][topic eltoo] before
+  the announcement of CTV, can substitute for CTV in many cases (but not
+  all), although it requires extra onchain vbytes to accomplish patterns
+  that CTV is optimized for.  At least one [variant of
+  SIGHASH_ANYPREVOUT][poinsot apo] has been proposed to extend the
+  number of CTV usecases covered, although it still requires extra
+  vbytes.
```

Comment:
The state of the art of LN-Symmetry has shown that CTV + CSFS + IKey is more bytewise efficient than APO for payment channels, although CTV + CSFS + IKey + APO is slightly more efficient.

cc @reardencode

Reply:
Leaving as-is.  This section isn't discussing the efficiency of APO vs CTV for payment channels, it's discussing APO as a generic alternative to CTV (which it isn't, not 100%).